### DISCUSSION TITLE
ModuleNotFoundError: No module named 'stlib3'

### DISCUSSION THREAD

**Discussion by oataga** [sofa-framework/sofa#3881] [Write / Run a simulation]
As in the title, once I try to launch the tripod-tuto.py I get the following message:

[ERROR] [SofaPython3::SceneLoader] Unable to completely load the scene from file 'C:\Users*******\Desktop\aaasofa\tripod-tuto.py'.
Python exception:
ModuleNotFoundError: No module named 'stlib3'

At:
C:\Users******\Desktop\aaasofa\tripod-tuto.py(1):
(219): _call_with_frames_removed
(848): exec_module
(6):

What should I do? I checked and the stlib3 folder is where it should be (/plugins/STLIB/lib/python3/site-packages/)

**Comment by damienmarchal** [comment]
Hello, in general this happens when the location where is the stlib3 directory is not accessible from python.
Can you check the directories where the python modules are loaded this with these lines
```python
import sys
print(sys.path)
```

**Reply by oataga** [reply]
![image](https://github.com/sofa-framework/sofa/assets/133664961/10304d28-c07c-4f05-bd1e-52078e201c1b)

**Reply by oataga** [reply]
I copied folders stlib3 and splib3 into Python\\Python38\\lib\\site-packages folder and it works c:

**Reply by jungjae01eng** [reply]
Add the STLIB to the PYTHONPATH in your .bashrc file.
